README.md

```diff
@@ -33,7 +33,7 @@ The following table provides a detailed list of all models materialized within t
 | [hubspot__contact_history](https://fivetran.github.io/dbt_hubspot/#!/model/model.hubspot.hubspot__contact_history) | Each record represents a change to a contact in Hubspot, with `valid_to` and `valid_from` information.               |
 | [hubspot__contact_lists](https://fivetran.github.io/dbt_hubspot/#!/model/model.hubspot.hubspot__contact_lists)   | Each record represents a contact list in Hubspot, enriched with metrics about email activities.                      |
 | [hubspot__deals](https://fivetran.github.io/dbt_hubspot/#!/model/model.hubspot.hubspot__deals)            | Each record represents a deal in Hubspot, enriched with metrics about engagement activities.                         |
-| [hubspot__deal_stages](https://fivetran.github.io/dbt_hubspot/#!/model/model.hubspot.hubspot__deal_stages)            | Each record represents a deal stage in Hubspot, enriched with metrics deal activities.                         |
+| [hubspot__deal_stages](https://fivetran.github.io/dbt_hubspot/#!/model/model.hubspot.hubspot__deal_stages)            | Each record represents when a deal stage changes in Hubspot, enriched with metrics deal activities.                         |
```

Comment:
great, i just [pushed](https://github.com/fivetran/dbt_hubspot/pull/132/commits/dec16d0208497240301c977a75a124510c1cfa39) the changes to this PR if you (or i can tag someone else in) could take a quick look 

note: i didn't _totally_ regen the docs because i realized i could directly change the model description in the manifest.json 🌚